install_config/topics/configuring_for_gce_registry-proc.adoc

```diff
@@ -0,0 +1,289 @@
+////
+Module included in the following assemblies:
+
+install_config/configuring_gce.adoc
+////
+
+= Configuring the {product-title} registry for GCP
+
+Google Cloud Platform (GCP) provides object cloud storage that {product-title}
+can use to store container images using the {product-title} container registry.
+
+For more information, see link:https://cloud.google.com/storage/docs/[`Cloud Storage` in the GCP documentation].
+
+.Prerequisites
+
+You must create the bucket to host the registry images before the installation.
+The following commands create a regional bucket using the configured service
+account:
+
+[source,yaml]
+----
+gsutil mb -c regional -l <region> gs://ocp-registry-bucket
+cat <<EOF > labels.json
+{
+  "ocp-cluster": "mycluster"
+}
+EOF
+gsutil label set labels.json gs://ocp-registry-bucket
+rm -f labels.json
+----
+
+[NOTE]
+====
+A bucket's data is automatically encrypted using a Google-managed key by
+default. To specify a different key to encrypt the data, see the
+https://cloud.google.com/storage/docs/encryption/[Data Encryption Options]
+available in GCP.
+====
+
+== Configuring {product-title} registry for GCP
+
+.Procedure
+
+To configure the Ansible inventory for the registry to use a `Google Cloud
+Storage` (`GCS`) bucket:
+
+[source,yaml]
+----
+[OSEv3:vars]
+# GCP Provider Configuration
+openshift_hosted_registry_storage_provider=gcs
+openshift_hosted_registry_storage_kind=object
+openshift_hosted_registry_replicas=1 <1>
+openshift_hosted_registry_storage_gcs_bucket=<bucket_name> <2>
+openshift_hosted_registry_storage_gcs_keyfile=<bucket_keyfile> <3>
+openshift_hosted_registry_storage_gcs_rootdirectory=<registry_directory> <4>
+----
+<1> The number of replicas to configure.
+<2> The bucket name to be used as registry storage.
+<3> The path on the installer host where the bucket's keyfile is located if
+using a custom key file to encrypt the data.
+<4> Directory used to store the data. `/registry` by default
+
+For more information, see link:https://cloud.google.com/storage/docs/[Cloud Storage in the GCP documentation].
+
+== Manually configuring {product-title} registry for GCP
+
+.Procedure
+
+To use GCP object storage, edit the registry’s configuration file and mount to the registry pod.
+
+. Take a default YAML configuration file as a base and replace the `filesystem`
+entry in the storage section with the GCP entry such as below. The resulting
+storage section may look like this:
++
```

Comment:
I'd say:

To use GCP object storage, the bucket and keyfile needs to be specified in the registry’s configuration file such as below. The resulting storage section may look like this:

Reply:
Changed to say:

Specify the bucket and keyfile in the `storage` section of a registry’s configuration file: